app/queue_consumers/tagging_updater.rb

```diff
@@ -0,0 +1,44 @@
+# TaggingUpdater
+#
+# To allow us to migrate to the new tagging infrastructure, we need to have
+# taggings be synced between panopticon and publishing-api. This class makes
+# sure that if an app sends `links` to the publishing-api, panopticon/contentapi
+# is also updated.
+class TaggingUpdater
+  def process(message)
+    content_item = message.payload
+
+    return message.ack unless content_item['publishing_app'].in?(Settings.apps_with_migrated_tagging)
+    return message.ack unless content_item['links']
+
+    slug = content_item.fetch('base_path').sub(/\A\//, '')
```

Comment:
I'd be inclined to pull this out into a separate method, and not do the early return on the `unless content_item['links']` check.  This would separate out the "filtering" of which messages we care about from the processing we do when we get a message we do care about.

So this would look something like
```
if content_item['links']
  process_content_item_with_links(content_item)
end
```

Separately - I think we will almost always have 'links' set, to at least an empty hash, in the messages we receive.  We might want to just make the code simpler by using `content_item.fetch('links', {})` to access the links, and avoid conditional paths.